pages/homePage.ts

```diff
@@ -0,0 +1,28 @@
+import { expect, Locator, Page } from "@playwright/test";
+import { config } from "../env.config";
+import { ProductsFilterFragment } from "./productsFiltersFragment";
+
+export class HomePage {
+  page: Page;
+  productsFilterFragment: ProductsFilterFragment;
+  card: Locator;
+
+  constructor(page: Page) {
+    this.page = page;
+    this.productsFilterFragment = new ProductsFilterFragment(page);
+    this.card = this.page.locator(".card");
+  }
+
+  async navigate(): Promise<void> {
+    await this.page.goto(`${config.weburl}`);
```

Comment:
You can update PW config to specify the baseURL - https://github.com/AronThomasMiller/PWProject/pull/2/files#diff-f679bf1e58e8dddfc6cff0fa37c8e755c8d2cfc9e6b5dc5520a5800beba59a92R32
And use just:
```suggestion
    await this.page.goto('/');
```